app/assets/javascripts/application.js

```diff
@@ -42,3 +42,42 @@ function show_notification_blood(msg){
 }
 
 
+function prepare_to_initialize(q) {
+  $(document).ready(function() {
+   google.maps.event.addDomListener(window, 'load',function () { initialize(q);});
+});
+}
+
+
+ function initialize(x) {
+
+  var input = document.getElementById(x);
+  var options = {
+    componentRestrictions: {country: "eg"}
+  };
+
+  var autocomplete = new google.maps.places.Autocomplete(input,options);
+
+ if (x == "searchTextField") {
```

Comment:
in app/views/users/registration/new.html.haml
can I change the name of class:'c-input' of field of searchTextField or this is related to another issue? 
and also can I change the class name of (f.text_field :birth_date) ,(f.text_field :lastdonation) in the same file?? to remove duplication of date picker.